embed/etcd.go

```diff
@@ -523,6 +523,10 @@ func (e *Etcd) serveClients() (err error) {
 }
 
 func (e *Etcd) serveMetrics() (err error) {
+	if e.cfg.Metrics == "extensive" {
```

Comment:
@Quentin-M can you define this as a const? now `extensive` string is used at two places.

Reply:
actually it seems that https://github.com/coreos/etcd/blob/2fb972847346d838b7395d9b5305ad45c124ff1b/etcdmain/etcd.go#L182 can be removed?

Reply:
Aren't those two independent mains / code paths? One for standalone etcd and one for embed. That was my understanding anyways. I noticed that the histograms were not available in the embed case, so made that PR.

Reply:
@Quentin-M 

binary one with main is a wrapper around embed one: see https://github.com/coreos/etcd/blob/2fb972847346d838b7395d9b5305ad45c124ff1b/etcdmain/etcd.go#L181-L186

Reply:
Ooooh I am blind as a monkey.. I should have used my brain for that PR.